src/Exceptions/PaymentException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace PodPoint\Payments\Exceptions;
+
+class PaymentException extends \Exception
```

Comment:
not keen on the naming on this, we are already within the `\Payments` namespace (the whole library is) so I don't think the word `Payment` is necessary here